pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "django-nps"
-version = "0.7.2"
+version = "0.8.0"
```

Comment:
Bump it to 1.0 - I think that the Py 3.8+ / Django 3.2+ version should be our 1.0 for anything that isn't already there?

Reply:
Ah ok! For future ones I'll ensure this is the case